var/spack/repos/builtin/packages/py-pynvtx/package.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyPynvtx(PythonPackage):
+    """A thin python wrapper for the nvToolsExt (NVTX) library, using
+    pybind11. This wrapper is meant to be as thin as possible -- so
+    only provides minimal support. Currently supported features are:
+
+    NVTX annotations: nvtxRangePushA and nvtxRangePop
+    Function decorator: PyNVTX.annotate
+    Automatic decorator generation PyNVTX.annotate_all_methods()"""
+
+    homepage = "https://github.com/JBlaschke/PyNVTX"
+    pypi = "PyNVTX/PyNVTX-0.3.3.tar.gz"
+
+    maintainers("DaxLynch")
+
+    version("0.3.3", sha256="8877b2d90bbf9d279d517a80a8f35a0a0a8179ebabf0729e806798a84bee6c72")
+
+    depends_on("py-setuptools@40.8:", type="build")
+    depends_on("py-pybind11", type=("build", "run"))
```

Comment:
Can you check the build log? I think this might also be a "link" dep, I see some header imports.

Reply:
Yes, will check

Reply:
Didn't see anything to indicate it is a link dep.